views/includes/error.ejs

```diff
@@ -0,0 +1,32 @@
+<% if (locals.error) { %>
+	<script type="text/javascript">
+		document.addEventListener('DOMContentLoaded', () => {
+			document.querySelectorAll('.hero.notification .delete')
+			.forEach(($delete) => {
```

Comment:
I do not think the $ are necessary in variables in this file

Reply:
Implicit norm in web... it tells you that you are playing with a HTMLElement-like object